README.md

```diff
@@ -1,2 +1,68 @@
-# cors-gate
-Enforce CORS server-side.
+cors-gate
+=========
+
+[![Build Status](https://travis-ci.org/mixmaxhq/cors-gate.svg?branch=master)](https://travis-ci.org/mixmaxhq/cors-gate)
+
+Connect-compatible middleware to selectively reject requests based on CORS rules.
+
+Install
+-------
+
+Run this in your project:
+
+```sh
+$ yarn add cors-gate
+```
+
+Test
+----
+
+```sh
+$ yarn test
+```
+
+Usage
+-----
+
+```js
+const express = require('express');
+const cors = require('cors');
+const corsGate = require('cors-gate');
+
+const app = express();
+
+app.use(cors({
+  origin: ['https://app.mixmax.com', 'https://api.mixmax.com'],
```

Comment:
What about in the tests like [here](https://github.com/mixmaxhq/cors-gate/pull/1/files#diff-aa089d3d3faceb6b5b0f1d81b944fe73R45)?